cvat/apps/engine/views.py

```diff
@@ -632,18 +633,22 @@ def _get_rq_response(queue, job_id):
     @action(detail=True, methods=['GET'], serializer_class=DataMetaSerializer,
         url_path='data/meta')
     def data_info(request, pk):
-        db_task = models.Task.objects.prefetch_related('data__images__related_files').select_related('data__video').get(pk=pk)
+        db_task = models.Task.objects.prefetch_related(
+            Prefetch('data', queryset=models.Data.objects.prefetch_related(
+                Prefetch('images', queryset=models.Image.objects.prefetch_related('related_files').order_by('frame'))
+            ))
+        ).get(pk=pk)
```

Comment:
Looks like ``select_related('data__video')`` is missed now. Don't we need it?
What is the difference between using string and objects as arguments?

Reply:
I've added `select_related('data__video')`, thanks!
With a simplified string argument it is not possible to specify a more complex subquery such as `order_by` or` filter` to be joined